src/components/base/misc/Icon.svelte

```diff
@@ -340,50 +293,16 @@
 		'x-circle',
 		'x-mark'
 	] as const;
-	export const icons = [...new Set([...featherIcons, ...heroIcons])];
-	type FeatherIconType = (typeof featherIcons)[number];
-	type HeroIconType = (typeof heroIcons)[number];
-	export type IconName = FeatherIconType | HeroIconType;
+	export const icons = [...new Set([...heroIcons])];
+	export type IconName = typeof icons;
 </script>
 
 <script lang="ts">
-	import { onMount } from 'svelte';
-
+	import CoreIcon from '@iconify/svelte';
 	export let name: IconName;
-	const isFeatherIcon = featherIcons.includes(name as FeatherIconType);
-
-	let component: any;
-	$: {
-		if (!isFeatherIcon) {
-			const pascalCase =
-				name
-					.split('-')
-					.map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-					.join('') + 'Icon';
-
-			//@ts-ignore FIXME
-		}
-	}
-
-	onMount(() => {
-		if (isFeatherIcon) {
-			//@ts-ignore
-			window?.feather?.replace();
-		}
-	});
 </script>
 
-{#if isFeatherIcon}
```

Comment:
Thankfully the new code is much more succint and I don't have to deal with this extra complexity